### DISCUSSION TITLE
错字

### DISCUSSION THREAD

**Discussion by dongrentianyu** [BramChen/tw5-docs#9] [General]
https://bramchen.github.io/tw5-docs/zh-Hans/#Behaviour%20of%20invoked%20variables%20depends%20on%20how%20the%20variable%20was%20declared

![image](https://github.com/BramChen/tw5-docs/assets/109711900/423c9adf-9a1c-4dae-a79f-153bbe897118)

**Comment by BramChen** [comment]
感恩 @dongrentianyu

**Reply by dongrentianyu** [reply]
客气了，我才要感恩你，没有你，就看不到中文文档了
